dev-lisp/sbcl/sbcl-2.1.10.22_44e0d89ff.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="A high performance Common Lisp compiler"
+DESCRIPTION="Steel Bank Common Lisp (SBCL) is a high performance Common \
+Lisp compiler. It is open source / free software, with a permissive license. \
+In addition to the compiler and runtime system for ANSI Common Lisp, it \
+provides an interactive environment including a debugger, a statistical \
+profiler, a code coverage tool, and many other extensions."
+HOMEPAGE="https://sbcl.org"
+COPYRIGHT="2002 Gerd Moellmann"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+srcGitRev="44e0d89ff14fcd8d599edd00668633c4772c26b9"
+SOURCE_URI="https://github.com/sbcl/sbcl/archive/$srcGitRev.tar.gz"
+SOURCE_DIR="sbcl-$srcGitRev"
+CHECKSUM_SHA256="b59dba5b0c4d8a599b60f29d79e29115f1dd73f3b5f305123ecf0cb1ef1a8212"
+PATCHES="sbcl-2.1.10.22_44e0d89ff.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+
+PROVIDES="
+	sbcl$secondaryArchSuffix = $portVersion
+	cmd:sbcl$secondaryArchSuffix = $portVersion
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+        java:environment >= 11
```

Comment:
Use 1 tab here (no spaces), same as in line 38, 52 and 53

Reply:
switched to tabs